### PR TITLE
Add distributed view in status dashboard

### PR DESCRIPTION
**Pull request by GowthamyVaseekaran**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/bean/KafkaTransportDetails.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package org.wso2.carbon.sp.jobmanager.core.bean;
+
+import java.util.List;
+
+/**
+ *
+ */
+public class KafkaTransportDetails {
```
Comment (pcnfernando) [inline components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/bean/KafkaTransportDetails.java]: Class comment missing

File: components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/ManagerDetails.java [hunk 2]
```diff
@@ -102,24 +72,22 @@ public boolean equals(Object o) {
             return false;
         }
         ManagerDetails managerDetails = (ManagerDetails) o;
-        return Objects.equals(this.host, managerDetails.host) &&
-                Objects.equals(this.port, managerDetails.port) &&
-                Objects.equals(this.haStatus, managerDetails.haStatus);
+        return
+                Objects.equals(this.haStatus, managerDetails.haStatus)&&
+                Objects.equals(this.groupId,managerDetails.groupId);
```
Comment (Anoukh) [inline components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/ManagerDetails.java]: Minor checkstyle issue

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+import Circle from "material-ui/svg-icons/av/fiber-manual-record";
+import {TableFooter} from "material-ui/Table/index";
+import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
+
+const dataConstants = {PAGE_LENGTH: 5};
+
+let currentPage = 1;
+
+/**
+ * class which manages Siddhi App list of a worker.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx]: Class level comment should be a sentence. Hence first letter (c) has be capital.

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+//Material UI
+import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from "material-ui/Table";
+import Pagination from "material-ui-pagination";
+import Circle from "material-ui/svg-icons/av/fiber-manual-record";
+import {TableFooter} from "material-ui/Table/index";
+import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
+
+const dataConstants = {PAGE_LENGTH: 5};
+
+let currentPage = 1;
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx]: Why this reside outside of the class?

Comment (GowthamyVaseekaran) [reply components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx]: That particular variable and const used inside render and componentWillMount

File: components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/bean/KafkaTransportDetails.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
+package org.wso2.carbon.sp.jobmanager.core.bean;
+
+import java.util.List;
+
```
Comment (pcnfernando) [inline components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/bean/KafkaTransportDetails.java]: Class comment missing

File: components/org.wso2.carbon.status.dashboard.core/pom.xml [hunk 6]
```diff
@@ -181,6 +181,35 @@
                     <basedir>${project.build.outputDirectory}/org/wso2/carbon/status/dashboard/core</basedir>
                 </configuration>
             </plugin>
+            <plugin><!-- Overridden from parent pom to exclude generated sources -->
```
Comment (pcnfernando) [inline components/org.wso2.carbon.status.dashboard.core/pom.xml]: don't override. include the plugin within your component. refer a checkstyle enabled component

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx (hunk 4 above)
Comment (this) [reply components/org.wso2.carbon.status.dashboard.web/src/distributedView/ParentAppTable.jsx]: then we can change this variable to a instance variable, right?

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.carbon.status.dashboard.core.model;
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java]: there should be a new line between license header and `package` statemen

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+/**
+ * Manager Deails bean class.
+ */
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java]: Unnecessary new line

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java (hunk 7 above)
Comment (this) [reply components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerDetails.java]: please fix in other places as well

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerOverView.java [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.carbon.status.dashboard.core.model;
+
+import org.wso2.carbon.status.dashboard.core.bean.ManagerClusterInfo;
+
+/**
+ * Class that gives an overview for manager node.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerOverView.java]: This comments indicates that this class has a method that gives an overview for manager node. But this is a modal class, which means it acts as a data model rather than an operator. Please change the class comment to correctly indicates its purpose.

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/WorkerOverview.java [hunk 10]
```diff
@@ -27,6 +27,7 @@
     private String statusMessage;
     private ServerDetails serverDetails;
 
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/WorkerOverview.java]: Why this new line?

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ChildApps.java [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * Class that contains the child siddhi app details.
+ */
+public class ChildApps {
+    private String parentAppName;
+    private String appName;
+    private String siddhiApp;
+
+    private ChildApps() {
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ChildApps.java]: No need for this empty constructor

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ManagerMetricsSnapshot.java [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+    private Long timeStamp;
+
+    public ManagerMetricsSnapshot(ManagerDetails workerMetrics, Long timeStamp) {
+        this.serverDetails = workerMetrics;
+        this.timeStamp = timeStamp;
+    }
+
+    public void updateRunningStatus(String status) {
+        serverDetails.setRunningStatus(status);
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ManagerMetricsSnapshot.java]: unnecessary new line

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ManagerMetricsSnapshot.java [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ */
+package org.wso2.carbon.status.dashboard.core.bean;
+
+import org.wso2.carbon.status.dashboard.core.model.ManagerDetails;
+
+/**
+ * Bean class for keeping the snapshot of manager status .
+ */
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ManagerMetricsSnapshot.java]: unnecessary new line

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/NodeConfigurationDetails.java [hunk 14]
```diff
@@ -24,21 +24,21 @@
 
 /**
  * WorkerConfigurationDetails bean class.
+ *
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/NodeConfigurationDetails.java]: unnecessary new line

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 15]
```diff
@@ -511,7 +520,7 @@ public List selectAppAggOverallMetrics(String metricsType, String workerId, long
      * @param timeInterval   time interval that needed to be taken.
      * @param metricTypeName metrics type name ex: memory,cpu
      * @param currentTime    current time in milliseconds.
-     * @return List<List<Object>> of metrics data because charts needed in that format
+     * @return List<List   <   Object>> of metrics data because charts needed in that format
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: code formatting issue; revert this change

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 16]
```diff
@@ -531,12 +540,13 @@ public List selectWorkerMetrics(String workerId, long timeInterval, String metri
...
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: code formatting issue; revert this change

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java [hunk 17]
```diff
@@ -88,6 +87,17 @@
     public static final String NEVER_REACHED = "Never Reached";
     public static final int DEFAULT_TIME_INTERVAL_MILLIS = 300000;
 
+    public static final String MANAGERID = "MANAGERID";
+    public static final String WORKERID = "WORKERID";
+    public static final String HOST = "HOST";
+    public static final String PORT = "PORT";
+    public static final String INTEGER_TYPE = "integerType";
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java]: what integer type?? be more descriptive. instead of suffixing, prefix them like `TYPE_INTEGER`

File: components/org.wso2.carbon.status.dashboard.web/public/css/dashboard.css [hunk 18]
```diff
@@ -173,3 +173,66 @@ body {
...
 }
 
+.deployed-apps-details {
+    color: white;
+    display: inline;
+}
+
+
+/*style sheet for kafka flow diagram*/
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/public/css/dashboard.css]: unnecessary new line

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx [hunk 19]
```diff
@@ -0,0 +1,352 @@
...
+    "hljs-string": {"color": "#f17b31"},
+    "hljs-keyword": {"color": "#f17b31"},
+    "hljs-selector-tag": {"color": "#f17b31"},
+    "hljs-addition": {"color": "#f17b31"},
+    "hljs-emphasis": {"fontStyle": "italic"},
+    "hljs-strong": {"fontWeight": "bold"}
+};
+
+/**
+ * class which manages Siddhi App specific details.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: first letter of a sentence should be capital

File: components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/ManagerDetails.java [hunk 20]
```diff
@@ -102,24 +72,22 @@ public boolean equals(Object o) {
...
-                Objects.equals(this.haStatus, managerDetails.haStatus);
+        return
+                Objects.equals(this.haStatus, managerDetails.haStatus)&&
+                Objects.equals(this.groupId, managerDetails.groupId);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(host, port, haStatus);
+        return Objects.hash(haStatus,groupId);
```
Comment (this) [inline components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/ManagerDetails.java]: code formatting issue: there should be a space after the comma

Comment (this) [reply components/org.wso2.carbon.sp.jobmanager.core/src/gen/java/org/wso2/carbon/sp/jobmanager/core/model/ManagerDetails.java]: please make sure that code formatting is properly done in other places as well

File: components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/impl/utils/Constants.java [hunk 21]
```diff
@@ -35,6 +36,14 @@
     public static final String LONG_TYPE = "longType";
     public static final String FLOAT_TYPE = "floatType";
     public static final String BOOL_TYPE = "booleanType";
+    public static final String ACTIVE = "Active";
```
Comment (this) [inline components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/impl/utils/Constants.java]: What is `ACTIVE`??
Use meaningful names for constants, prefix them appropriately.

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/api/RFC3339DateFormat.java [hunk 22]
```diff
@@ -24,6 +25,9 @@
 import java.text.FieldPosition;
 import java.util.Date;
 
+/**
+ * RFC3339DateFormat.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/api/RFC3339DateFormat.java]: expand this class level javadoc

File: components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerOverView.java [hunk 23]
```diff
@@ -0,0 +1,79 @@
...
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.status.dashboard.core.model;
+
+import org.wso2.carbon.status.dashboard.core.bean.ManagerClusterInfo;
+
+/**
+ * Which are needed only to over view page.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerOverView.java]: `Which are`?? Please make sure javadoc comments are full sentences.

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ParentSiddhiApp.java [hunk 24]
```diff
@@ -0,0 +1,66 @@
...
+/**
+ * Class that contains the parent siddhi application details.
+ */
+public class ParentSiddhiApp {
+    private String parentAppName;
+    private String groupName;
+    private String appName;
+    private String id;
+
+    private ParentSiddhiApp(){
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ParentSiddhiApp.java]: Unnecessary empty constructor.
No need to put this as javac will add the default for you.

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ParentSummaryDetails.java [hunk 25]
```diff
@@ -0,0 +1,75 @@
...
+ * This class contains the summary details of each parent siddhi application.
+ */
+public class ParentSummaryDetails {
+    private List<String> groups = new ArrayList<>();
+    private int childApps = 0;
+    private List<String> usedWorkerNode = new ArrayList<>();
+    private List<String> deployedChildApps = new ArrayList<>();
+    private List<String> unDeployedChildApps = new ArrayList<>();
+
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/bean/ParentSummaryDetails.java]: extra new line, please remove.

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 26]
```diff
@@ -552,7 +563,7 @@ public List selectWorkerAggregatedMetrics(String workerId, long timeInterval, St
      * @param workerId     source id of the metrics
      * @param timeInterval time interval that metrics needed to be taken.
      * @param currentTime  current time
-     * @return List<List<Object>> of metrics data because charts needed in that format
+     * @return List<List   <   Object>> of metrics data because charts needed in that format
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: incorrect code formatting, please revert this change

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 27]
```diff
@@ -572,7 +583,7 @@ public List selectWorkerThroughput(String workerId, long timeInterval, long curr
...
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: incorrect code formatting, please revert this change

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java [hunk 28]
```diff
@@ -88,6 +88,17 @@
     public static final String NEVER_REACHED = "Never Reached";
     public static final int DEFAULT_TIME_INTERVAL_MILLIS = 300000;
 
+    public static final String MANAGERID = "MANAGERID";
+    public static final String WORKERID = "WORKERID";
+    public static final String HOST = "HOST";
+    public static final String PORT = "PORT";
+    public static final String INTEGER_DATA_TYPE = "integerType";
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java]: instead of suffixing, prefix the constants names e.g. `DATA_TYPE_INTEGER`.

Comment (this) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java]: also `DATA_TYPE` of what?

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java [hunk 29]
```diff
@@ -88,6 +88,17 @@
     public static final String NEVER_REACHED = "Never Reached";
     public static final int DEFAULT_TIME_INTERVAL_MILLIS = 300000;
 
+    public static final String MANAGERID = "MANAGERID";
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java]: What is `MANAGERID`, `WORKERID`, `HOST`, `PORT`??
use meaningful names for constants e.g. `WORKER_JVM_BLOCKED_THREADS_COUNT`.

File: components/org.wso2.carbon.status.dashboard.web/public/css/dashboard.css [hunk 30]
```diff
@@ -173,3 +173,65 @@ body {
...
+.node rect {
+    stroke: #bbb;
+    fill: #333;
+    stroke-width: 2px;
+    height: 60px;
+}
+
+.node-content {height: 50px;
+    line-height: 30px;
+    /*width: 400px;*/
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/public/css/dashboard.css]: please remove commented-out codes

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx [hunk 31]
```diff
@@ -0,0 +1,352 @@
...
+                                        display: 'inline-block',height:30}}>
+                                         <span className='legend-color' style={{ width: 15,
+                                            height: 15,
+                                            float: 'left',
+                                            margin: 5,
+                                            backgroundColor:'red'}}></span>
+
+                                        <span className='legend-text' style={{ lineHeight: 2,
+                                            fontSize: 12}}> Kafka-Topics</span>
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: please remove unnecessary new line

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewAppThumbnail.jsx [hunk 32]
```diff
@@ -0,0 +1,174 @@
...
+                <Snackbar contentStyle={messageBoxStyle} bodyStyle={this.state.messageStyle} open={this.state.showMsg}
+                          message={this.state.message} autoHideDuration={4000}
+                          onRequestClose={() => {
+                              this.setState({showMsg: false, message: ""})
+                          }}/>
+            </div>
+        );
+    }
+
+}
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewAppThumbnail.jsx]: please add file-ending new line

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx [hunk 33]
```diff
@@ -0,0 +1,352 @@
...
+import SyntaxHighlighter from "react-syntax-highlighter";
+
+//App Components
+import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
+import Header from "../common/Header";
+import ParentAppTable from "./ParentAppTable";
+//Material UI
+import HomeButton from "material-ui/svg-icons/action/home";
+import {
+    Card,
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: why this is formatted like this? In other places you have import statements like 
```js
import {CardActions, Dialog, FlatButton, GridList, GridTile} from "material-ui";
```

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx [hunk 34]
```diff
@@ -0,0 +1,352 @@
...
+    CardHeader,
+    CardText,
+    Dialog,
+    Divider,
+    FlatButton,
+    Snackbar, RaisedButton
+} from "material-ui";
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: Don't use `"` for strings, always use `'`. Please fix in other places where you have used `"`.

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx [hunk 35]
```diff
@@ -0,0 +1,352 @@
...
+//import DashboardUtils from "../utils/DashboardUtils";
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: commented-our code!

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx (hunk 34 above)
Comment (this) [reply components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx]: this is due to JS code style guideline

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx [hunk 36]
```diff
@@ -0,0 +1,311 @@
...
+                        statusMessage: "Unknown error occurred! : " + JSON.stringify(error.response.data)
+                    });
+                }
+            }
+        });
+
+
+    }
+
+
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx]: unnecessary new line

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx [hunk 37]
```diff
@@ -0,0 +1,311 @@
...
+    componentWillUnmount() {
+        clearInterval(this.state.interval);
+
+    }
+
+    componentWillMount() {
+        let that = this;
+        // this.initAutoSync();
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx]: commented-out code

File: components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx [hunk 38]
```diff
@@ -0,0 +1,311 @@
...
+                        <Card style={{
+                            width: 700, high: '100%', marginTop: '10%', marginLeft: '33%', backgroundColor: '#1a1a1a',
+                            borderColor: '#f17b31', borderRadius: 2, borderBottomColor: '#f17b31'
+                        }}>
+                            <CardText style={{borderBottom: '1px solid #AE5923', borderTop: '1px solid #AE5923'}}>
+                                <FormPanel title={""} width={650}>
+                                    <div style={errorContainerStyles}>
+                                        <i class="fw fw-security fw-inverse fw-5x"></i>
+                                        <h1 style={errorTitleStyles}>Page Forbidden!</h1>
+                                        <text style={errorMessageStyles}>You have no permission to access this page.
```
Comment (this) [inline components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx]: code formatting

Comment (GowthamyVaseekaran) [reply components/org.wso2.carbon.status.dashboard.web/src/distributedView/DistributedViewOverview.jsx]: done
